### PR TITLE
Build images in ram

### PR DESCRIPTION
**Pull request by SunilMohanAdapa**
I saw significant speed up.  I was able to successfully builld and amd64 image. However, I didn't test it.

### CODE REVIEW

**Comment by SunilMohanAdapa** [comment]
Image seems to be booting well baring the problem with users first-setup.

**Comment by SunilMohanAdapa** [comment]
I rebased the patch onto current master and tested again by building a fresh image. I was able to finish the Plinth first-run wizard (due to fixes in Plinth 0.20)

File: freedommaker/builder.py [hunk 1]
```diff
@@ -125,6 +132,25 @@ def _get_image_base_name(self):
...
+
+        If building to RAM is disabled, append .temp to the final file name and
+        return it.
+
+        """
+        if not self.arguments.build_in_ram:
+            return self.image_file + '.temp'
+
+        self.ram_directory = tempfile.TemporaryDirectory()
+        self._run(['mount', '-t', 'tmpfs', 'tmpfs', self.ram_directory.name])
```
Comment (jvalleroy) [inline freedommaker/builder.py]: When running freedommaker as normal user, I get a build error:
```
subprocess.CalledProcessError: Command '['mount', '-t', 'tmpfs', 'tmpfs', '/tmp/tmpylh69806']' returned non-zero exit status 1
```
Running this command directly shows the reason as:
```
mount: only root can use "--types" option
```
Could "sudo" be added here?

**Comment by SunilMohanAdapa** [comment]
I have pushed two more changes (that may be squashed into the first commit) that will fix the sudo properly and also another cleanup problem when build fails.

**Comment by jvalleroy** [comment]
Merged.

**Pull request closed without merging**